### ISSUE TITLE
Understanding CohortAlleleFreq in merged vs unmerged files

### ISSUE THREAD

**Issue by gabyrech** [smail-lab-cmh/ga4k-sv-finder#1]
First, thank you for your efforts on making this data available! 

I am interested on Insertions (INS) AF in your data set but when I was deciding on whether to use the merged or unmerged files I found something I would need you help with. According to the documentation, the 6th column represent the CohortAlleleFreq (AF), which  in the case of the 'unmerged' file I think makes sense. For instance, if I focus on the INS with the stars (***) below I can see the AF match with my understanding of the maths:

Unmerged file:
```
$ grep -v "CohortAlleleFreq" 07172024_unmerged.tsv | sort -k1,1 -k2,2n | grep "INS"| less
chr1    32023   32023   64      INS     0.0016  0       5       0.0032  .       .
chr1    54753   54753   50      INS     0.0029  2       7       0.0045  .       .
chr1    59598   59598   315     INS     0.0022  1       6       0.0038  .       .     *** (a)
chr1    59598   59598   316     INS     0.001   0       3       0.0019  .       .     *** (b)
chr1    66251   66251   212     INS     0.0026  3       5       0.0032  .       .
chr1    66251   66251   213     INS     0.0026  2       6       0.0038  .       .
chr1    66475   66475   267     INS     0.001   0       3       0.0019  .       .
```
Total Alleles = 1566 * 2 = 3132 
CohortAlleleFreq = Alleles (Homozygotes+VariantCount) / Total Alleles
*** (a): 6 + 1 = 7 / 3132 = 0.002234993614304 ---> Match with CohortAlleleFreq (0.0022)
*** (b): 3 + 0 = 3 / 3132 = 0.00095785440613 ---> Match with CohortAlleleFreq (rounded to 0.001)

However, when I look at the merged data set I found a couple of things I dont understand:
```
$ grep -v "CohortAlleleFreq" 07172024_survivor_merged.tsv | sort -k1,1 -k2,2n | grep "INS" | less
chr1    10252   10252   2609    INS     0.00128 2       2       0.001   .       .
chr1    32023   32023   64      INS     0.0016  0       5       0.003   .       .
chr1    59598   59598   315     INS     0.00383 1       14      0.009   .       .     *** (c)
chr1    66251   66251   212     INS     0.0099  8       23      0.015   .       .
chr1    83873   83873   54      INS     0.01086 9       25      0.016   .       .
```
First, could you explain how is it possible to have '14' as VariantCount (8th column) after merging?  I just see 9 (6+3) from the unmerged file. 

Second, my estimation of the CohortAlleleFreq doesn't match to the reported in the merged file:
*** (c): 14 + 1 = 15 / 3132 = 0.004789272030651 ---> NO MATCH, 0.00383 is reported. 
In fact, the value reported in the CohortAlleleFreq column in the merged file (0.00383) suspiciously match with the CohortVariantFreq of one of the entries in the unmerged file (*** (a) 0.0038). Am I doing something wrong or misinterpreting something? 

Thanks!
Gabriel

**Comment by cas-lemaster** [comment]
The problem is with how SURVIVOR merged files are created. Since we have nearly 1600 vcfs that go into the merged vcf, these files can get pretty big. Thus, the parsing script we use on that, compared to the one we used on the unmerged files is a bit different. In the SURVIVOR merged dataset, the calculation of allele frequency is dependent on genotyped variants. In that calculation, we exclude any homozygous refs or incompletes, such as “0/0”, “./.” , or “./1”. This means that the allele frequency and the homozygote count are accurate, but the variant count and subsequent variant frequency are inflated since they’re purely counting the number of files the variant is found, regardless of genotype. With your specific example and our individual files, I found that that 11 samples had the merged INS at CHR1:59598-59598. Not 14. Three individuals had “./.” genotypes. Thus, for 11 individuals where one is homozygous, that would be (11 + 1)/3132 = 0.00383. 

So, you might now wonder, “but I only saw 9 in the unmerged dataset, where did 11 come from?” Though there are 11 individuals composing the merged INS, 2 of those individuals have unique INS lengths. So, while we can report the 6 with one length and the 3 with another length, the two with unique lengths cannot be reported because the variant+length is held by single individuals, which falls under the number of individuals required to make it publicly available per ethics guidance at CMH. In the merged dataset, more variants can compose a single allele frequency. We offered two datasets as a way for researchers interested in different perspectives to have access with their agenda in mind but unfortunately that can lead to some discrepancies between datasets due to their complexity.

Thus, we have updated the merged tsv in the repo, replacing the inflated values in the VariantCount and CohortVariantFreq columns with "NA".

Thanks for your help in identifying this issue.
